### ISSUE TITLE
Improper Python source encoding definitions.

### ISSUE THREAD

**Issue by scotmatson** [mpicard/pyvas#13]
https://www.python.org/dev/peps/pep-0263/

Noticed the use of encoding definitions in each file is a bit varied and doesn't have uniformity across the project and in some cases violates the standard.

Usage:
```# coding: utf-8``` or ```# -*- coding: utf-8 -*-```

**Comment by mpicard** [comment]
Thanks for letting me know! I’ll update it today

**Comment by mpicard** [comment]
@scotmatson I'm always open to PRs! I need some here for sure. I haven't been doing any openvas stuff at work so I have zero time to work on it. Feel free to add whatever is missing. I obviously have a lot of API surface in OMPv8/9 that isn't covered. I also would love to setup CI with docker image of openvas but haven't gotten around to it

**Comment by scotmatson** [comment]
I recently returned to OpenVAS after being sidelined with another project so I will likely be looking towards Pyvas again soon.

Recently setup a PoC of a distributed OpenVAS network via AWS EC2 & Portainer using https://github.com/mikesplain/openvas-docker as our foundation. Mike did an excellent job putting together an up-to-date docker image.

**Comment by mpicard** [comment]
Yeah haven't had any time to setup a container for openvas but i'd love to run the unit tests in travis against a container. I think i'd make the project more open to contributions if you didn't have to setup openvas locally to write/run unit tests. Or we could use docker locally in addition.
